application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -4603,6 +4603,42 @@ input TransferPreferencesInput {
   slack: Duration
 }
 
+"A collection of selectors for what routes/agencies should be included in or excluded from the search."
+input TransitFilterInput {
+  """
+  A list of selectors for what routes/agencies should be allowed during search.
+    
+  In order to be excluded a route/agency has to match with at least one selector.
+    
+  An empty list or a list containing `null` is forbidden.
+  """
+  exclude: [TransitFilterSelectInput!]
+  """
+  A list of selectors for what routes/agencies should be excluded during the search.
+    
+  If route/agency matches with at least one selector it will be included.
+    
+  An empty list or a list containing `null` is forbidden.
+  """
+  include: [TransitFilterSelectInput!]
+}
```

Comment:
We should document the priority of include/exclude. If something is included but also excluded, the exclude is the one that rules.